ui/components/app/snaps/keyring-snap-removal-warning/keyring-snap-removal-warning.tsx

```diff
@@ -10,12 +10,12 @@ import {
   ButtonSize,
   ButtonVariant,
   Modal,
-  ModalContent,
-  ModalHeader,
   ModalOverlay,
   Text,
   TextField,
 } from '../../../component-library';
+import { ModalContent } from '../../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../../component-library/modal-header/deprecated';
 import {
   BlockSize,
   Display,
```

Comment:
Screenshots showing no visual changes to UI

### Before
<img width="1512" alt="Screenshot 2023-12-10 at 2 42 54 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/2709acfe-3266-49d0-9370-ba6eff1daa6b">

### After
<img width="1512" alt="Screenshot 2023-12-10 at 2 43 04 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/23860e11-2333-41dd-ad42-78754fea64b3">